company-ycmd.el

```diff
@@ -436,7 +436,7 @@ If CB is non-nil, call it with candidates."
        (ycmd-running?)
        (or (not (company-in-string-or-comment))
            (company-ycmd--in-include))
-       (or (company-grab-symbol-cons "\\.\\|->\\|::" 2)
+       (or (company-grab-symbol-cons "/\\|\\.\\|->\\|::" 2)
```

Comment:
Can you put the path separator at the end of the regex?

Also what about Windows?